examples/tutorials/job_cli.ipynb

```diff
@@ -0,0 +1,1552 @@
+    "\n",
+    "You can also directly edit this file or simply use\n",
+    "\n",
+    "```\n",
+    "nvflare config -jt ../../job_templates. \n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "1bea478d-42fb-4223-8155-3c996699a052",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! nvflare config -jt ../../job_templates "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9a7b708-40e9-4f4f-beec-b83d0e893a0b",
+   "metadata": {},
+   "source": [
+    "Now we can list again with job_templates directory argument"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "49bab293-7fef-476f-8aa9-f2b2868a0fb3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "The following job templates are available: \n",
+      "\n",
+      "------------------------------------------------------------------------------------------------------------------------\n",
+      "  name            Description                                                  Controller Type      Client Category     \n",
+      "------------------------------------------------------------------------------------------------------------------------\n",
+      "  sag_cross_np    scatter & gather and cross-site validation using numpy       server               client executor     \n",
+      "  sag_lightning   scatter & gather workflow using lightning                    server               client_api          \n",
+      "  sag_pt          scatter & gather workflow using pytorch                      server               client_api          \n",
+      "  stats_df        FedStats: tabular data with pandas                           server               stats executor      \n",
+      "------------------------------------------------------------------------------------------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "! nvflare job list_templates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10720b0b-af4c-4d71-a751-2d5c301eb05a",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "once we found the job template that fits our needs, we can use the job template name to create a new job folder\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ecbd0cc7-f410-457e-a6bd-d4c999182850",
+   "metadata": {},
+   "source": [
+    "### Create a job folder\n",
+    "\n",
+    "Since our code is written in pytorch and we would like to try FedAvg algorithm using Scatter & Gather (SAG) workflow, the job template **\"sag_pt\"** is what we are looking for. We will use this template to create our job folder. \n",
+    "\n",
+    "Creat a job folder is to create a job directory that contains the job configuration.  First, we create job folder and intend to be modified, without specify our code.\n",
```

Comment:
We create a job folder that contains the base job configuration from the template, and can then modify it as desired.